deploy/cloud/README.md

```diff
@@ -0,0 +1,22 @@
+# Cloud Deployment
+
+**Motivation**
+Provide an easy and deterministic way to setup latest cloud environment so it can be monitored and used properly.
+
+**Prerequisite**
+* [terraform](https://www.terraform.io/)
+
+**How To**
+1. Create an [API token](https://cloud.elastic.co/deployment-features/keys) from your cloud console account.
+    1.1 use the token `export EC_API_KEY={TOKEN}`
+2. run `cd deploy/cloud`
+3. run `terraform init`
+4. run `terraform apply --auto-approve` to create the environment from the latest version (the latest version is vary in cloud/regions combinations). To set a specific version use `-var="stack_version=8.5.1"`
+
+To delete the environment:
```

Comment:
consider giving examples how to connect to the deployed environment

like: using the cloud login / `terraform output -json` to reveal the passwords

Reply:
Removed the sensitive keyword.